### ISSUE TITLE
The `generate-vscode-jars` profile should not include lemminx jar and its dependencies

### ISSUE THREAD

**Issue by fbricon** [eclipse/lemminx-maven#529]
When running `mvn clean verify -DskipTests -Dinvoker.skip=true -Pgenerate-vscode-jars`, the lemminx jar and its dependencies (including tests), should be excluded.

**Comment by vrubezhny** [comment]
@fbricon then how are we  going run lemminx-maven in vscode-xml?

**Comment by fbricon** [comment]
Vscode-xml already provides lemminx and it's dependencies. We only need lemminix-maven and its own specific dependencies

**Comment by vrubezhny** [comment]
@fbricon  I think this is still a question in case of vscode-xml is set to use a lemminx binary

**Comment by fbricon** [comment]
no, the lemminx jar is always shipped in vcode-xml, whether you use the binary or not.

**Comment by dasTanmoy0096** [comment]
vscode-uber-jars also contains some extre dependencies such as junit, which is only a test scope dependency..?  But yeah the specific artifacts to be included needs to be cleaned out more thoroughly.
